engine/executor/config.go

```diff
@@ -50,6 +50,9 @@ type Config struct {
 	AdvertiseAddr string `toml:"advertise-addr" json:"advertise-addr"`
 
 	Labels map[string]string `toml:"labels" json:"labels"`
+	// ExclusivelyDeployed means only one executor is deployed in the container/machine,
+	// which means it can consume almost all the resources of the container/machine.
+	ExclusivelyDeployed bool `toml:"exclusively-deployed" json:"exclusively-deployed"`
```

Comment:
What about using a more explicit config, such as `EnableGCTuning`, it is difficult to determine whether enable gc tuner is safe even under exclusively deployment.